spec/unit/iiif/v3/presentation/sequence_spec.rb

```diff
@@ -1,98 +1,176 @@
 describe IIIF::V3::Presentation::Sequence do
 
   describe '#required_keys' do
-    it 'accumulates from the superclass' do
-      expect(subject.required_keys).to eq %w{ type items }
+    %w{ type items }.each do |k|
+      it k do
+        expect(subject.required_keys).to include(k)
+      end
     end
   end
 
-  let(:subclass_subject) do
-    Class.new(IIIF::V3::Presentation::Sequence) do
-      def initialize(hsh={})
-        hsh = { 'type' => 'a:SubClass' }
-        super(hsh)
-      end
+  describe '#prohibited_keys' do
+    it 'contains the expected key names' do
+      keys = described_class::CONTENT_RESOURCE_PROPERTIES +
+        described_class::PAGING_PROPERTIES +
+        %w{
+          nav_date
+          content_annotations
+        }
+      expect(subject.prohibited_keys).to include(*keys)
```

Comment:
because the method is implemented as `super + `  I wanted to leave room for super to add whatever.